opensearch/src/main/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/relevance/WildcardQuery.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+package org.opensearch.sql.opensearch.storage.script.filter.lucene.relevance;
+
+import org.opensearch.index.query.QueryBuilders;
+import org.opensearch.index.query.WildcardQueryBuilder;
+
+/**
+ * Lucene query that builds wildcard query.
+ */
+public class WildcardQuery extends SingleFieldQuery<WildcardQueryBuilder> {
+  /**
+   *  Default constructor for WildcardQuery configures how RelevanceQuery.build() handles
+   * named arguments.
+   */
+  public WildcardQuery() {
+    super(FunctionParameterRepository.WildcardQueryBuildActions);
+  }
+
+  private String convertSqlWildcardToLucene(String text) {
+    return text.replace('%', '*')
+               .replace('_', '?');
+  }
```

Comment:
How would this account for escaped special characters?
In query like `SELECT * FROM t WHERE f LIKE '\%'`, search string needs to be converted to '`%'` not `'\*'`.

Reply:
Implemented in 0b9752ca994a334de8e9bbc6bd7ac49fadd37944 however, needs more tests